### PR TITLE
Update local_setup.md

### PR DESCRIPTION
**Pull request by BrendenBarbour**
Added changes found by Carolina to get her machine functional.
Added lines on how to start the vm/test local setup.
Fixed line ending error during terraform apply.

### CODE REVIEW

**Comment by rak5216** [comment]
@Josh-Joseph i think you will have to make approval since we can't replace you with me

File: docs/local_setup.md [hunk 1]
```diff
@@ -12,7 +12,8 @@ This software easily allows integration with GitHub version control and seamless
 
 Git Bash is a terminal that we use to rectify Unix line endings within windows machines. Git should have been installed with GitHub Desktop. If not, follow the steps below.
 
-Follow the link here to download Git. Once GitBash is installed, use the following command: `git config --global core.autocrlf false`
+Follow the link here to download Git. Once GitBash is installed, use the following command within that console: `git config --global core.autocrlf false`
+Additionally open Github Desktop and enter the following lines: `git config core.eol lf', 'git config core.autocrlf "input"`.
```
Comment (Josh-Joseph) [inline docs/local_setup.md:16]: Two nitpick-y bits here:
- would add some wording to make it clear this optional and only if you use github desktop
- a beginning user won't know where in github desktop to enter the following lines. going to have to fill in that middle of step of "go to whatever menu it's called -> select terminal (or whatever they call it in the menu)"

**Review by Josh-Joseph — APPROVED** [review]
looks good - just the one line needs a bit of editing!

**Comment by Josh-Joseph** [comment]
happy to review this but just for future reference you should be able to change/replace reviewers by clicking the little sprocket thing next to reviewers and selecting the person that has been selected (with the check mark). It should then remove them.

**Pull request merged**